integ-test/src/test/java/org/opensearch/sql/sql/NestedIT.java

```diff
@@ -366,4 +365,105 @@ public void test_nested_in_where_as_predicate_expression_with_relevance_query()
     assertEquals(1, result.getInt("total"));
     verifyDataRows(result, rows(10, "a"));
   }
+
```

Comment:
Can you add tests with `SELECT nested(message.*) as <alias>` and something like `SELECT upper(nested(message.*))`?

Reply:
And `SELECT nested(message.*) FROM ... WHERE nested(message.info) == 'a'`

Reply:
I can add an IT test with WHERE clause. Alias isn't supported!

Reply:
Test says it is just ignored. A bug? A feature?
```
opensearchsql> SELECT nested(message.*) as pewpew from nested-type;
fetched rows / total rows = 8/8
+------------------------------+-----------------------------+------------------------+
| nested(message.author)       | nested(message.dayOfWeek)   | nested(message.info)   |
|------------------------------+-----------------------------+------------------------|
| e                            | 1                           | a                      |
| f                            | 2                           | b
...
```
Maybe : create a GH ticket for future work.